pkgs/development/python-modules/sox/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchPypi
+, python
+, buildPythonPackage
+, pythonOlder
+, numpy
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "sox";
+  version = "1.4.1";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    hash = "sha256-sPLRNpJFC4ic0/ZhJ+lvgBlC7CqsW7IWU9/RUODXEFU=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    typing-extensions
+  ];
+
+  doCheck = false;
```

Comment:
I am working on this but I got a lot of tests looking for files in `/build/sox-1.4.1/tests/data` but are not found.
Test code is using `os.path.dirname(__file__)` which I assume returns `/build/sox-1.4.1/tests` which seems to not be the correct value.

Since I am not a python dev *and* new to nix packaging, I have no idea on how to fix that.

Any help is welcome.